src/main/java/duke/commands/CommandDeadline.java

```diff
@@ -0,0 +1,22 @@
+package duke.commands;
+
+import duke.DukeException;
+import duke.TaskManager;
+
+public class CommandDeadline {
+    public static void processNewDeadline(String command, TaskManager taskManager)
+            throws DukeException.IllegalDeadlineFormatException, DukeException.IllegalDeadlineDateException {
+        int spacePosition = command.indexOf(" ");
+        int dividerPosition = command.indexOf("/");
```

Comment:
Perhaps you can consider replacing the delimiters with constants, to eliminate magic literals.